docs/Networking.md

```diff
@@ -130,4 +130,4 @@ ws.onclose = (e) => {
 };
 ```
 
-Your app can now display all sorts of data and you may soon need to organize this content into several screens. To manage the transition between these screens, you will need to learn about [navigators](docs/using-navigators.html).
```

Comment:
This paragraph served as the segue to the Using Navigators Basics guide, which has been removed. Can you delete the entire paragraph instead?